src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14016,10 +14015,14 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                                 state, max(RefTSat, Pipe_Te_assumed + Modifi_SHin), max(min(Pipe_Pe_assumed, RefPHigh), RefPLow), RoutineName);
 
                             if (Pipe_h_IU_out_i > Pipe_h_IU_in) {
+                                Real64 min_speed_capacity =
+                                    this->CoffEvapCap * this->RatedEvapCapacity * CurveValue(state, this->OUCoolingCAPFT(1), T_discharge, T_suction);
                                 Pipe_m_ref_i = (state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).TotalCoolLoad(NumTU) <= 0.0)
                                                    ? 0.0
-                                                   : (state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).TotalCoolLoad(NumTU) /
-                                                      (Pipe_h_IU_out_i - Pipe_h_IU_in));
+                                                   // refrigerant flow rate do not get smaller when the compressor needs to cycle.
+                                                   : max(min_speed_capacity,
+                                                         (state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).TotalCoolLoad(NumTU)) /
+                                                             (Pipe_h_IU_out_i - Pipe_h_IU_in));
```

Comment:
This may be the very change that corrects several issues with this model.

Reply:
This fixed the super high cooling COP problem: from previous maximum of 158 to about 12 now.